nomad/structs/funcs_test.go

```diff
@@ -366,7 +381,10 @@ func TestAllocsFit(t *testing.T) {
 	require.NoError(err)
 	require.False(fit)
 
-	// Sanity check the used resources
+	// Check the used resources are expected values
+	// TODO: This comment is not super useful, but neither was
+	//       the original. It would be better if someone
+	//       explained the invariant that this check validates.
```

Comment:
I can't do this as a suggestion because of the way GH broke the diff, but this whole class of comment looks like it's really about the allocation that's tested before it. I'd suggest it get reduced to something like:

```go
	// Should not fit second allocation, and the used resources should match the first alloc
	fit, _, used, err = AllocsFit(n, []*Allocation{a1, a1}, nil, false)
	require.NoError(err)
	require.False(fit)
	require.EqualValues(2000, used.Flattened.Cpu.CpuShares)
	require.EqualValues(2048, used.Flattened.Memory.MemoryMB)
```

Or you could even just drop the "Check the used resources are expected values" comment entirely and move the two lines up under the lines before them. It's reasonably clear what's being checked in that context I think.

Reply:
Okay. I will tidy up those comments really quick.  Thanks for the suggestions!